Makefile

```diff
@@ -14,19 +21,14 @@ endif
 
 all: manager
 
-# Create a local docker registry, start kinD cluster, and install Orkestra
+# Create a local docker registry, start kind cluster, and install Orkestra
 dev:
-	bash scripts/kind-with-registry.sh
+	make kind-create
```

Comment:
You can just move kind-create above one line as a pre-run for dev instead of invoking `make` again.
`dev: kind-create`, unless I am missing something?

Reply:
If the idea from the make dev to have the full steps, then we should keep creating kind as sub-step.